[2023-09-18 23:05:26,418] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:26] "[33mDELETE /deletar_cliente?nome=leandro HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:27,855] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:27] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:28,754] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:28] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:29,327] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:29] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:29,431] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:29] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:29,738] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:29] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:29,994] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:29] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:41,904] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:41] "OPTIONS /deletar_cliente?nome=leandro HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:41,909] WARNING del_cliente() L128  Erro ao deletar cliente #'leandro', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 23:05:41,909] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:41] "[33mDELETE /deletar_cliente?nome=leandro HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:49,083] WARNING add_cliente() L42   Adicionando cliente de nome: 'aa' - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L42  
[2023-09-18 23:05:49,100] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:49] "POST /cadastrar_cliente HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:50,815] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:50] "OPTIONS /deletar_cliente?nome=leandro HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:05:50,819] WARNING del_cliente() L128  Erro ao deletar cliente #'leandro', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 23:05:50,819] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:05:50] "[33mDELETE /deletar_cliente?nome=leandro HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:06:08,481] WARNING add_cliente() L42   Adicionando cliente de nome: 'bb' - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L42  
[2023-09-18 23:06:08,481] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:06:08] "POST /cadastrar_cliente HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:06:10,455] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:06:10] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:08:20,341] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:08:20] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:09:08,803] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:09:08] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:13:30,295] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:13:30] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:16:36,505] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:16:36] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:17:08,485] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:17:08] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:25:50,290] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:25:50] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:25:55,639] WARNING add_cliente() L42   Adicionando cliente de nome: 'dd' - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L42  
[2023-09-18 23:25:55,657] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:25:55] "POST /cadastrar_cliente HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:27:52,119] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:27:52] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:27:56,035] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:27:56] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:27:56,144] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:27:56] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:27:56,274] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:27:56] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:31:14,131] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:31:14] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:31:22,543] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:31:22] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:31:57,515] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:31:57] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:31:58,375] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:31:58] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:09,490] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:32:09] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:10,221] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:32:10] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:14,820] WARNING add_cliente() L42   Adicionando cliente de nome: 'ee' - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L42  
[2023-09-18 23:32:14,840] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:32:14] "POST /cadastrar_cliente HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:17,167] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:32:17] "OPTIONS /deletar_cliente?nome=ee HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:17,183] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:32:17] "DELETE /deletar_cliente?nome=ee HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:19,070] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:32:19] "OPTIONS /deletar_cliente?nome=leandro HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:19,073] WARNING del_cliente() L128  Erro ao deletar cliente #'leandro', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
[2023-09-18 23:32:19,073] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:32:19] "[33mDELETE /deletar_cliente?nome=leandro HTTP/1.1[0m" 404 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:25,463] INFO _log() L187  127.0.0.1 - - [18/Sep/2023 23:32:25] "OPTIONS /deletar_cliente?nome=leandro HTTP/1.1" 200 - - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\env\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-18 23:32:25,468] WARNING del_cliente() L128  Erro ao deletar cliente #'leandro', Cliente não encontrado na base :/ - call_trace=C:\Users\leand\Desktop\sprint 1\mvp_app_api\app.py L128 
